docs/project-handoff.md

```diff
@@ -1,22 +0,0 @@
-# Project Handoff
-
-This is a quick guide for Dr. Decker on how to set up a new team with the project
-
-## Accounts
-
-New team members' accounts will need to be added to the following:
-
-1. [Slack](https://app.slack.com/client/TMU604V63) (I recommend you keep the same slack group instead of making a new one, because useful information is posted in the [#setup](https://app.slack.com/client/TMU604V63/C011H12GV7F) channel)
-2. Google Drive [team drive](https://drive.google.com/drive/folders/0AA8xzw8iWvvHUk9PVA)
-3. [Trello board](https://trello.com/b/9VtTK5fV/rit-senior-project-monuments-and-memorials) (I recommend you keep the same Trello board because backlogged items are there for the next team to work on)
-4. [GitHub team](https://github.com/orgs/team-monumental/people)
-
-### Accounts to be Aware of
-
-There are a few accounts that are shared by all team members, so you don't need to add them to the account but you should make them aware of their existence
-
-1. [Zoho Mail](https://mail.zoho.com/) - This is the third party we use to have three free email addresses at our custom domain
-2. [Google Developer Console](https://console.cloud.google.com/google/maps-apis/overview?pli=1&project=monuments-268618&folder=&organizationId=&supportedpurview=project) - This is where the Google Maps API usage is managed
-3. [Amazon AWS Console](https://console.aws.amazon.com/console/home?region=us-east-1) - This is where all of our AWS services (EC2, S3, SES) are managed
-4. [Namecheap.com](https://namecheap.com) - This is where the domain name is registered and managed
-
```

Comment:
What did you do with this document?

Reply:
https://docs.google.com/document/d/1gl9kQkQhuNIow0u-Y0L3KEmxa5xKswklmC-PpJOR5ME